packages/account-v2/src/pages/LoginHistory/LoginHistory.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Table } from '@deriv-com/ui';
+
+type TRowData = {
+    action?: string;
+    browser?: string;
+    datetime?: string;
+    ipAddress?: string;
+    status?: string;
+};
+const columnOrder = ['datetime', 'action', 'browser', 'ipAddress', 'status'] as const;
+
+const headers = {
+    datetime: 'Date and Time',
+    action: 'Action',
+    browser: 'Browser',
+    ipAddress: 'IP Address',
+    status: 'Status',
+};
```

Comment:
```suggestion
const headers = {
    action: 'Action',
    browser: 'Browser',
    datetime: 'Date and Time',
    ipAddress: 'IP Address',
    status: 'Status',
};
```

due to eslint issue `Expected object keys to be in natural insensitive ascending order. 'action' should be before 'datetime'.eslint[sort-keys](https://eslint.org/docs/rules/sort-keys)`